ftml/src/render/html/element/footnotes.rs

```diff
@@ -24,26 +24,53 @@ pub fn render_footnote(log: &Logger, ctx: &mut HtmlContext) {
     info!(log, "Rendering footnote reference");
 
     let index = ctx.next_footnote_index();
+    let id = &format!("{}", index);
+
+    let footnote_string = ctx.handle().get_message(log, ctx.language(), "footnote");
+    let label = &format!("{} {}.", footnote_string, index);
```

Comment:
Yeah, I thought about that. I didn't want to complicate things. This might be fixable on the JS side using the `Intl` standard library, but I'm not sure if it handles indexing like this (it probably does). So yeah, we'll have to wait on this until we actually know what we need to do for a particular language.